ODIN_II/regression_test/benchmark/task/keywords/always/synthesis_result.json

```diff
@@ -0,0 +1,522 @@
+        "Total Node": 0
+    },
+    "always/always_or_event/k6_frac_N10_frac_chain_mem32K_40nm": {
+        "test_name": "always/always_or_event/k6_frac_N10_frac_chain_mem32K_40nm",
+        "architecture": "k6_frac_N10_frac_chain_mem32K_40nm.xml",
+        "verilog": "always_or_event.v",
+        "exit": 1,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 32.9,
+        "exec_time(ms)": 67.7,
+        "synthesis_time(ms)": 0.7,
+        "Latch Drivers": 0,
+        "Pi": 2,
+        "Po": 2,
+        "logic element": 0,
+        "latch": 0,
+        "Adder": 0,
+        "Multiplier": 0,
+        "Memory": 0,
+        "Hard Ip": -1,
+        "generic logic size": 4,
+        "Longest Path": 2,
+        "Average Path": 2,
+        "Estimated LUTs": 0,
+        "Total Node": 0
+    },
+    "always/always_or_event/k6_N10_40nm": {
+        "test_name": "always/always_or_event/k6_N10_40nm",
+        "architecture": "k6_N10_40nm.xml",
+        "verilog": "always_or_event.v",
+        "exit": 1,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 14.6,
+        "exec_time(ms)": 10,
+        "synthesis_time(ms)": 0.8,
+        "Latch Drivers": 0,
+        "Pi": 2,
+        "Po": 2,
+        "logic element": 0,
+        "latch": 0,
+        "Adder": -1,
+        "Multiplier": -1,
+        "Memory": -1,
+        "Hard Ip": -1,
+        "generic logic size": 6,
+        "Longest Path": 2,
+        "Average Path": 2,
+        "Estimated LUTs": 0,
+        "Total Node": 0
+    },
+    "always/always_or_event/k6_N10_mem32K_40nm": {
+        "test_name": "always/always_or_event/k6_N10_mem32K_40nm",
+        "architecture": "k6_N10_mem32K_40nm.xml",
+        "verilog": "always_or_event.v",
+        "exit": 1,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 32,
+        "exec_time(ms)": 68.5,
+        "synthesis_time(ms)": 0.8,
+        "Latch Drivers": 0,
+        "Pi": 2,
+        "Po": 2,
+        "logic element": 0,
+        "latch": 0,
+        "Adder": -1,
+        "Multiplier": 0,
+        "Memory": 0,
+        "Hard Ip": -1,
+        "generic logic size": 6,
+        "Longest Path": 2,
+        "Average Path": 2,
+        "Estimated LUTs": 0,
+        "Total Node": 0
+    },
+    "always/always_or_event/no_arch": {
+        "test_name": "always/always_or_event/no_arch",
+        "architecture": "n/a",
+        "verilog": "always_or_event.v",
+        "exit": 1,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 11.3,
+        "exec_time(ms)": 3.1,
+        "synthesis_time(ms)": 1.4,
+        "Latch Drivers": 0,
+        "Pi": 2,
+        "Po": 2,
+        "logic element": 0,
+        "latch": 0,
+        "Adder": -1,
+        "Multiplier": -1,
+        "Memory": -1,
+        "Hard Ip": -1,
+        "generic logic size": -1,
+        "Longest Path": 2,
```

Comment:
if we don't exit 0 everything should be -1 (or non existant would actually be better I believe... maybe a todo?)